src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -545,6 +542,101 @@ void testCreateVirtualSchemaWithoutIgnoreErrorsPropertyThrowsException() throws
                         + "`TIMESTAMP WITH LOCAL TIME ZONE` in Virtual Schemas can produce an incorrect results"));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = { "SELECT *", "SELECT BOOL_COL, VARCHAR_COL, DECIMAL_COL" })
+    void testSelectAllColumnsWithExplicitSelectList(String select) throws SQLException {
+        final Table table = this.sourceSchema.createTable("TEST_TABLE", "BOOL_COL", "BOOLEAN", "VARCHAR_COL",
+                "VARCHAR(100)", "DECIMAL_COL", "DECIMAL(18,0)");
+        createVirtualSchemaWithoutSelectListProjectionCapability();
+        final String explainVirtual = getExplainVirtual(
+                select + " FROM " + getVirtualTableName(this.virtualSchema, table));
+        assertThat(explainVirtual, containsString("SELECT \"TEST_TABLE\".\"BOOL_COL\", \"TEST_TABLE\".\"VARCHAR_COL\", "
```

Comment:
I would strongly advice against using `EXPLAIN VIRTUAL` for testing.
What you want to do with the integration tests is to make sure that a sub-assembly of system components operates correctly.
So you want to see them as a black box, for decoupling. This means that if at some point the remote database change it's implementation, you don't have to worry about it, as your tests deals with the public API (this is true for most kind of tests BTW).
As I see it, the purpose of `EXPLAIN VIRTUAL` is mostly for debugging, so it's prone to create coupling.
Instead, I would advice to check the results of executing such queries, instead of the generated `SQL` string.
One scenario were using `EXPLAIN VIRTUAL` might be worth is to tests, for example, performance, but it's not the case here.

Reply:
same comment applies to the following tests.

Reply:
We can't test the changed behavior with the result of the query. Because the result is the same with the old code and after the changes. We can have these tests or don't have any tests at all which is not good. I don't want to test it manually each time.